<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Error</class>
 <widget class="QDialog" name="Error">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>202</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>253</width>
    <height>144</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 42, 47);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
